Rakefile

```diff
@@ -3,3 +3,23 @@ require "logstash/devutils/rake"
 task :default do
   system('rake -T')
 end
+
+task :gradle => "gradle.properties" do
```

Comment:
We should have consistent naming for this task.  In [beats input](https://github.com/logstash-plugins/logstash-input-beats/blob/master/lib/tasks/build.rake#L5) it is called ```vendor```. I don't have a naming preference, just a preference for consistency.